resolwe_bio/management/commands/generate_samples.py

```diff
@@ -147,15 +157,18 @@ def create_data(self, reads_name='seq_reads', annotated=False, rseed=None):
         mouse_genes = os.path.join(test_files_path, 'mouse_genes.tab.gz')
 
         # crate reads data object
+        started = timezone.now()
```

Comment:
Dates are always tricky. Please check if this work in the resolwe-bio test Django project that does not use time zones and in the genialis-base test Django project that uses time zones.